From e8967c31a649c512aeab6e186d52db87a4845954 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Thu, 27 Sep 2018 00:23:32 +0300
Subject: [PATCH 14/16] common: label /sys/devices/virtual/graphics as
 sysfs_graphics

* To avoid granting global sysfs permissions to the init scripts

Change-Id: I7fca4d7aed84ef9612cff6649e3ebcec4de20d4d
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 common/genfs_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/genfs_contexts b/common/genfs_contexts
index cac3200..e57b6fe 100644
--- a/common/genfs_contexts
+++ b/common/genfs_contexts
@@ -16,6 +16,7 @@ genfscon sysfs /devices/soc/soc:qcom,memlat-cpu2/devfreq u:object_r:sysfs_devfre
 genfscon sysfs /devices/soc/soc:qcom,memlat-cpu4/devfreq u:object_r:sysfs_devfreq:s0
 genfscon sysfs /devices/soc/soc:qcom,mincpubw/devfreq u:object_r:sysfs_devfreq:s0
 genfscon sysfs /kernel/wcd_cpe0 u:object_r:sysfs_audio:s0
+genfscon sysfs /devices/virtual/graphics u:object_r:sysfs_graphics:s0
 genfscon sysfs /devices/virtual/thermal u:object_r:sysfs_thermal:s0
 genfscon sysfs /devices/virtual/kgsl/kgsl/proc u:object_r:sysfs_kgsl_proc:s0
 genfscon debugfs /kgsl/proc     u:object_r:kgsl_debugfs:s0
-- 
2.17.1

